resources/assets/tconstruct/book/en_US/smeltery/structure.json

```diff
@@ -2,10 +2,10 @@
   "title":"",
   "text": [
     {
-      "text": "The smeltery can be any size from 3x3 to 11x11, with larger structures increasing both the number of items that can smelt and the liquid capacity. The structure requires a floor, but does not require a ceiling or a frame."
```

Comment:
Whoops, typo from copy paste. I added that line to the seared furnace and tinker tank but forgot to make it 2 for the smeltery...